### DISCUSSION TITLE
@walletconnect/modal-react-native/src/components/ModalHeader.tsx: Unexpected token, expected ","

### DISCUSSION THREAD

**Discussion by outlawcode** [WalletConnect/walletconnect-monorepo#3818] [Developer Support]
[Error: TransformError SyntaxError: /Users/mac/Apps/hotromua/htm_app/node_modules/@walletconnect/modal-react-native/src/components/ModalHeader.tsx: Unexpected token, expected "," (1:24)

I got this error, please help!

**Comment by glitch-txs** [comment]
This need more information - please read https://github.com/orgs/WalletConnect/discussions/3457
